File: user-rest-service-1.0.2/generated-code/csharp/src/main/csharp/io/swagger/Model/ShowcaseDatatypePrimitives.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;



namespace io.swagger.Model {
  public class ShowcaseDatatypePrimitives {
    

    
    public string SingleByte { get; set; }

    

    
    public int? TinyNumber { get; set; }

    

    
    public int? Counter { get; set; }

    

    
    public long? Id { get; set; }

    

    
    public long? BudgetFloat { get; set; }

    

    
    public double? Budget { get; set; }

    

    
    public bool? Deleted { get; set; }

    

    
    public string SingleCharacter { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class ShowcaseDatatypePrimitives {\n");
      
      sb.Append("  SingleByte: ").Append(SingleByte).Append("\n");
      
      sb.Append("  TinyNumber: ").Append(TinyNumber).Append("\n");
      
      sb.Append("  Counter: ").Append(Counter).Append("\n");
      
      sb.Append("  Id: ").Append(Id).Append("\n");
      
      sb.Append("  BudgetFloat: ").Append(BudgetFloat).Append("\n");
      
      sb.Append("  Budget: ").Append(Budget).Append("\n");
      
      sb.Append("  Deleted: ").Append(Deleted).Append("\n");
      
      sb.Append("  SingleCharacter: ").Append(SingleCharacter).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: user-rest-service-spring-rest-client-java/src/main/java/io/swagger/client/model/ShowcaseDatatypePrimitives.java
package io.swagger.client.model;



import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;



/**
 * 
 **/
@ApiModel(description = "")
public class ShowcaseDatatypePrimitives  {
  
  private String singleByte = null;
  private Integer tinyNumber = null;
  private Integer counter = null;
  private Long id = null;
  private Double budget = null;
  private Boolean deleted = null;
  private String singleCharacter = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("singleByte")
  public String getSingleByte() {
    return singleByte;
  }
  public void setSingleByte(String singleByte) {
    this.singleByte = singleByte;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("tinyNumber")
  public Integer getTinyNumber() {
    return tinyNumber;
  }
  public void setTinyNumber(Integer tinyNumber) {
    this.tinyNumber = tinyNumber;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("counter")
  public Integer getCounter() {
    return counter;
  }
  public void setCounter(Integer counter) {
    this.counter = counter;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("id")
  public Long getId() {
    return id;
  }
  public void setId(Long id) {
    this.id = id;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("budget")
  public Double getBudget() {
    return budget;
  }
  public void setBudget(Double budget) {
    this.budget = budget;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("deleted")
  public Boolean getDeleted() {
    return deleted;
  }
  public void setDeleted(Boolean deleted) {
    this.deleted = deleted;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("singleCharacter")
  public String getSingleCharacter() {
    return singleCharacter;
  }
  public void setSingleCharacter(String singleCharacter) {
    this.singleCharacter = singleCharacter;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class ShowcaseDatatypePrimitives {\n");
    
    sb.append("  singleByte: ").append(singleByte).append("\n");
    sb.append("  tinyNumber: ").append(tinyNumber).append("\n");
    sb.append("  counter: ").append(counter).append("\n");
    sb.append("  id: ").append(id).append("\n");
    sb.append("  budget: ").append(budget).append("\n");
    sb.append("  deleted: ").append(deleted).append("\n");
    sb.append("  singleCharacter: ").append(singleCharacter).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}


